src/sentry/ingest/types.py

```diff
@@ -10,3 +10,4 @@ class ConsumerType(StrEnum):
     Attachments = "attachments"  # consumes events with attachments ( from the Attachments topic)
     Transactions = "transactions"  # consumes transaction events ( from the Transactions topic)
     Feedback = "feedback"  # consumes user feedback ( from the ingest-feedback-events topic)
+    Profiles = "profiles"
```

Comment:
Nit: a comment like the others mainly for the name of the topic it consumes from might be handy

Reply:
I thought I could use the generic `IngestConsumerFactory` but I can't, this shouldn't be here.